src/sentry/charts/endpoints.py

```diff
@@ -0,0 +1,24 @@
+import os.path
+from typing import Union
+from django.views import static
+
+import sentry
+
+from django.http import (
+    FileResponse,
+    Http404,
+    HttpResponse,
+    HttpResponseNotModified,
+    HttpRequest,
+)
+
+
+CONFIG_DIR = os.path.abspath(
+    os.path.join(os.path.dirname(sentry.__file__), "../../config/chartcuterie")
```

Comment:
```suggestion
    os.path.join(os.path.dirname(sentry.__file__), "..", "..", "config", "chartcuterie")
```

Reply:
👍

Reply:
@EvanPurkhiser I think it's fine as is tbh. Unsure if Sentry ever works in filesystems like Windows.